packages/lesswrong/server/migrations/2021-12-13-updateQuadraticVotes.ts

```diff
@@ -64,11 +64,13 @@ registerMigration({
       }
     }
 
-    Object.keys(votesByUserId).forEach(async (userId) => {
+    for (let userId in votesByUserId) {
       let totalUserPoints = 0 
       const user = usersByUserId[userId][0]
 
-      votesByUserId[userId].forEach(async vote => {
+      for (let vote of votesByUserId[userId]) {
+        if (!vote.qualitativeScore) return
```

Comment:
Should this be `continue`?

Reply:
If not maybe add a quick docstring

Reply:
It damn well should. Thanks for the catch.